specifyweb/frontend/js_src/lib/components/FormSliders/RecordSet.tsx

```diff
@@ -412,7 +416,37 @@ function RecordSet<SCHEMA extends AnySchema>({
                 })
             : undefined
         }
-        onClone={(newResource): void => go(totalCount, 'new', newResource)}
+        onClone={(resources: RA<SpecifyResource<SCHEMA>>): void => {
+          go(totalCount, 'new', resources[0]);
+          /*
+           * );
+           * Bulk carry when there are multiple resources
+           */
+          if (resources.length > 1) {
+            const sortedResources = Array.from(resources).sort(
+              sortFunction((r) => r.id)
+            );
+            loading(
+              createNewRecordSet(
+                sortedResources.map((resource) => resource.id)
+              ).then(async () => {
+                const startingResourceCatNumber =
+                  serializeResource(
+                    sortedResources[0] as SpecifyResource<CollectionObject>
+                  ).catalogNumber ?? '0';
```

Comment:
actually, you might be better off using `defined(startingResourceCatNumber)`. Same for endingResourceCatNumber

Reply:
Oh also, why do we need to serializeResource? Why not `sortedResources[0].get('catalognumber')`

Reply:
same error with defined